protobufs/livekit_rtc_v2.proto

```diff
@@ -77,6 +104,20 @@ message ConnectRequest {
   // will overwrite if the same key is in the token
   // will not delete keys from token if there is a key collision and this sets that key to empty value
   map<string, string> participant_attributes = 4;
+
+  repeated AudioTrack audio_tracks = 5;
+  repeated VideoTrack video_tracks = 6;
```

Comment:
These tracks are added as part of `ConnectRequest`. Ideally, these should always be added and never queued. But, this differs from sending `AddTrackRequest` later. How should we think about it?